test/e2e/upgrade/alert/alert.go

```diff
@@ -83,8 +83,9 @@ func (t *UpgradeTest) Test(f *framework.Framework, done <-chan struct{}, upgrade
 	// period by verifying Watchdog alert has been in firing state
 	watchdogQuery := fmt.Sprintf(`count_over_time(ALERTS{alertstate="firing",alertname="Watchdog", severity="none"}[%dm])`, alertCheckSleepMinutes)
 
-	// Query to check for any critical severity alerts that have occurred within the last alertPeriodCheckMinutes
-	criticalAlertQuery := fmt.Sprintf(`count_over_time(ALERTS{alertname!~"Watchdog|AlertmanagerReceiversNotConfigured|KubeAPILatencyHigh",alertstate="firing",severity="critical"}[%dm]) >= 1`, alertPeriodCheckMinutes)
+	// Query to check for any critical severity alerts that have occurred within the last alertPeriodCheckMinutes.
```

Comment:
when you squash can you keep this in its own commit but put the other changes in this file into the other commit?  Basically 2 - one for this, one for your test?

Reply:
Done